modules/1_prepare/variables.tf

```diff
@@ -23,12 +23,14 @@ variable "cluster_id" {
   type = string
 
   validation {
-    condition     = length(var.cluster_id) <= 14
-    error_message = "Length cannot exceed 14 characters when combined with cluster_id_prefix."
+    condition     = length(var.cluster_id) < 32
+    error_message = "Length cannot exceed 32 characters when combined with cluster_id_prefix."
   }
 }
 variable "bastion" {}
 
+variable "name_prefix" {}
+
```

Comment:
Seems you have increased the cluster_id length to 32 without considering `name_prefix` that has the zone info. We will need to see if `name_prefix` (cluster_id+zone) is <= 32

Reply:
Done.